packages/clis/generator/commands/graph.ts

```diff
@@ -66,9 +66,9 @@ export async function handler(args: BuilderArguments<typeof builder>) {
     includeHidden: false,
   });
 
-  const graph = generateGraph(tsMapData);
+  const graph = generateGraph(tsMapData, args.map);
```

Comment:
🤔 maybe `args.map` can be part of `tsMapData`. Then it wouldn't be possible to call, e.g., `generateGraph(atsMapData, 'europe')`.